newIDE/app/src/InstancesEditor/InstancesMover.js

```diff
@@ -11,28 +11,24 @@ export default class InstancesResizer {
     this.options = options;
   }
 
-  _roundXPosition(x) {
-    if (!this.options.snap || !this.options.grid || this.options.gridWidth <= 0)
+  _roundXPosition(x, noGridSnap) {
+    if (!this.options.snap || !this.options.grid || this.options.gridWidth <= 0 || noGridSnap)
       return Math.round(x);
 
     return (
       Math.round((x - this.options.gridOffsetX) / this.options.gridWidth) *
-        this.options.gridWidth +
+      this.options.gridWidth +
       this.options.gridOffsetX
     );
   }
 
-  _roundYPosition(y) {
-    if (
-      !this.options.snap ||
-      !this.options.grid ||
-      this.options.gridHeight <= 0
-    )
+  _roundYPosition(y, noGridSnap) {
+    if (!this.options.snap || !this.options.grid || this.options.gridHeight <= 0 || noGridSnap)
```

Comment:
Can you run prettier on the files? In VSCode, once you have prettier extension installed, press Cmd+Shift+P and enter "Format Document".

Prettier is an automatic code formatter, all the code is formatted by this :)

Reply:
Will do.